src/Microsoft.SqlTools.ServiceLayer/ServiceHost.cs

```diff
@@ -171,6 +172,140 @@ public void RegisterInitializeTask(InitializeCallback callback)
                 });
         }
 
+        internal async Task HandleCapabilitiesRequest(
+            CapabilitiesRequest initializeParams, 
+            RequestContext<CapabilitiesResult> requestContext)            
+        {
+            await requestContext.SendResult(
+                new CapabilitiesResult
+                {
+                    Capabilities = new DmpServerCapabilities
+                    {
+                        ProtocolVersion = "1.0",
```

Comment:
I would personally be in favor of making these `readonly`/`const` at the top of the file, but it's obviously not 100% necessary.